var/spack/repos/builtin/packages/paraver/package.py

```diff
@@ -32,9 +32,9 @@ class Paraver(Package):
         is expressed on its input trace format.  Traces for parallel MPI,
         OpenMP and other programs can be genereated with Extrae."""
     homepage = "http://www.bsc.es/computer-sciences/performance-tools/paraver"
-    url      = "http://www.bsc.es/ssl/apps/performanceTools/files/paraver-sources-4.5.3.tar.gz"
+    url      = "http://www.bsc.es/ssl/apps/performanceTools/files/paraver-sources-4.6.2.tar.gz"
 
-    version('4.5.3', '625de9ec0d639acd18d1aaa644b38f72')
```

Comment:
Paraver provides only latest version for download. So I assume people will add version if they see if it's available.

I can add note saying "Paraver provides only latest version for download, don't keep/add older versions".